app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">MovieFiner</string>
+    <string name="title">Title</string>
+    <string name="date">2022-12-14</string>
```

Comment:
You need an extension to convert the date format like 12 dec,2022

Reply:
https://stackoverflow.com/questions/57402045/how-to-format-in-kotlin-date-in-string-or-timestamp-to-my-preferred-format